packages/orbit-components/src/Loading/index.tsx

```diff
@@ -49,7 +50,12 @@ const Loader = ({ type, customSize, title }) => {
 
   if (isCircledIcon) {
     return (
-      <div className="flex items-center justify-center">
+      <div
+        className="flex items-center justify-center"
+        aria-hidden={ariaHidden}
+        role="img"
```

Comment:
When type=INLINE_LOADER and title is not filled in (it's optional), we're getting a11y violation:
![image](https://github.com/user-attachments/assets/c799712e-de81-41ca-879a-118066a783e4)

I'm not sure if this variant will happen, but I think it could be nice to have this case covered. 🤔